src/containers/Reaction/helpers/helpers.js

```diff
@@ -0,0 +1,13 @@
+export const isLikedByUser = (likedUsers, userId) => {
+  const isLiked = likedUsers.filter(
```

Comment:
Hi @Danielshow. Could you make changes to the naming of the file `i.e helper.js`. It seems vague.